app/src/containers/ListOfNews/index.js

```diff
@@ -59,13 +57,32 @@ class ListOfNews extends React.Component {
             });
 
         callback();
-    }
+  */ 
+
+      //axios.get('news.json').then(response=>{
+        //this.sort((a, b) =>{return a.title - b.title});
+        //this.setState({ newsList: response});
+      
+      //})
+
+
+      fetch('news.json')
```

Comment:
You are getting data from your local test data file here. If this goes to production, think about what will happen. The front-end won't fetch data from the news api. You probably do not want that to happen.